### PR TITLE
[jingoo] Release version v1.3.1

### PR DESCRIPTION
**Pull request by tategakibunko**

### CODE REVIEW

File: packages/jingoo/jingoo.1.3.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/tategakibunko/jingoo/issues"
+dev-repo: "git+https://github.com/tategakibunko/jingoo.git"
+license: "BSD-3"
+synopsis: "Template engine almost compatible with Jinja2(python template engine)"
+build: [make "all"]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.4.0"}
```
Comment (kit-ty-kate) [inline packages/jingoo/jingoo.1.3.1/opam]:
```suggestion
  "dune" {>= "1.4.0"}
```
Dune shouldn't be tagged as a build dependency (see #14266)

File: packages/jingoo/jingoo.1.3.0/opam [hunk 2]
```diff
@@ -4,14 +4,14 @@ authors: "Masaki WATANABE <lambda.watanabe@gmail.com>"
...
-license: "BSD-3-Clause"
+license: "BSD-3"
 synopsis: "Template engine almost compatible with Jinja2(python template engine)"
 build: [make "all"]
 install: [make "install"]
 remove: [make "uninstall"]
 depends: [
   "ocaml" {>= "4.04.0"}
-  "dune" {>= "1.4.0"}
+  "dune" {build & >= "1.4.0"}
```
Comment (kit-ty-kate) [inline packages/jingoo/jingoo.1.3.0/opam]:
```suggestion
  "dune" {>= "1.4.0"}
```
Dune shouldn't be tagged as a build dependency (see #14266)

File: packages/jingoo/jingoo.1.3.0/opam [hunk 3]
```diff
@@ -4,14 +4,14 @@ authors: "Masaki WATANABE <lambda.watanabe@gmail.com>"
 homepage: "https://github.com/tategakibunko/jingoo"
 bug-reports: "https://github.com/tategakibunko/jingoo/issues"
 dev-repo: "git+https://github.com/tategakibunko/jingoo.git"
-license: "BSD-3-Clause"
+license: "BSD-3"
```
Comment (kit-ty-kate) [inline packages/jingoo/jingoo.1.3.0/opam]:
```suggestion
license: "BSD-3-Clause"
```
This has been changed by https://github.com/ocaml/opam-repository/pull/14773 to normalize the license names.

File: packages/jingoo/jingoo.1.3.1/opam [hunk 4]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Masaki WATANABE <lambda.watanabe@gmail.com>"
+authors: "Masaki WATANABE <lambda.watanabe@gmail.com>"
+homepage: "https://github.com/tategakibunko/jingoo"
+bug-reports: "https://github.com/tategakibunko/jingoo/issues"
+dev-repo: "git+https://github.com/tategakibunko/jingoo.git"
+license: "BSD-3"
```
Comment (kit-ty-kate) [inline packages/jingoo/jingoo.1.3.1/opam]:
```suggestion
license: "BSD-3-Clause"
```

File: packages/jingoo/jingoo.1.3.1/opam [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+synopsis: "Template engine almost compatible with Jinja2(python template engine)"
+build: [make "all"]
+install: [make "install"]
+remove: [make "uninstall"]
```
Comment (kit-ty-kate) [inline packages/jingoo/jingoo.1.3.1/opam]:
```suggestion
```
This field is not necessary since opam 2.0

**Comment by tategakibunko** [comment]
Thank you for your review!

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**